R/serialize.R

```diff
@@ -66,6 +68,28 @@ serialize_parse_data_flat <- function(pd_flat) {
         collapse = "")) %>%
     .[["text_ws"]] %>%
     strsplit("\n", fixed = TRUE) %>%
-    .[[1L]]
+    .[[1L]] %>%
+    trimws(which = "right")
```

Comment:
Because we don't do it elsewhere  anymore and I think it's easiest done here. I think we once had a transformer for it but that does not work if we do not want to touch indention (next topic after this PR is merged) because there, the spaces after the token are the spaces after the token and the line breaks (so spaces = indention for last token on a line).  In your initial version of styler, we trimmed the initial text and replaced tabs with two white spaces. Why did we remove it there again?